README.md

```diff
@@ -0,0 +1,12 @@
+# Accelerator
+Accelerator replaces the default SRCDS crash handler with one that is a lot more reliable and uploads the crash reports to a community-accessible processing backend.
+This is useful since Valve's processing system discards crash reports from modded servers - and they still receive a representative sample of crash reports for actual game issues.
+
+The processing backend ([Throttle](https://crash.limetech.org/)) parses crash reports to extract useful information, and in the case of common issues affecting many servers where manual analysis has been done, tags them with helpful notices with information on resolving the cause of the crash.
```

Comment:
@asherkin mentioned in the discord that my previous call out over analyses was wrong, it is the correct term. However let's fix this marathon of a sentence.
```markdown
The server's crash report is sent to the processing backend service ([Throttle](https://crash.limetech.org/)) for analysis.
Throttle will compare the crash to other existing crash reports and provide a link to an overview page of the matching crash signature and could include information on resolving the cause of the crash.
```